src/style.css

```diff
@@ -1,20 +1,27 @@
-
 body {
   margin: 20px 60px;
-  font-family: 'Comic Neue', cursive;
-  border: 5px solid black;
+  font-family: 'Open Sans', sans-serif;
+  border-radius: 10px;
   min-width: 600px;
-  max-width: 768px;
+  max-width: 1200px;
+  background-color: #59beda;
+  background-image: url(./assets/mario.jpg);
```

Comment:
- [ ] [Optional] There is a warning on the browser as `asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).` so what I suggest is that please decrease the size of the background image or you can remove it and use the next one.